scripts/webpack/js-bundle-factory.js

```diff
@@ -62,9 +65,18 @@ class JsBundleFactory {
         library,
       },
       plugins = [],
+      tsConfigFilePath = path.resolve(__dirname, '../../tsconfig.json'),
     }) {
     chunks = chunks || this.globber_.getChunks({inputDirectory, filePathPattern});
 
+    const babelLoader = {
+      loader: 'babel-loader',
+      options: {
+        cacheDirectory: true,
+        presets: [['es2015', {modules: false}]],
```

Comment:
Isn't this picked up from the `babel` property in our `package.json`? We have other babel configuration there as well which makes me concerned whether all of it is being picked up correctly...

Reply:
I removed the options here. package.json's options do no have cacheDirectory. Definition reads:

```
cacheDirectory: Default false. When set, the given directory will be used to cache the results of the loader. Future webpack builds will attempt to read from the cache to avoid needing to run the potentially expensive Babel recompilation process on each run. If the value is blank (loader: 'babel-loader?cacheDirectory') or true (loader: 'babel-loader?cacheDirectory=true'), the loader will use the default cache directory in node_modules/.cache/babel-loader or fallback to the default OS temporary file directory if no node_modules folder could be found in any root directory.```

I think this can be applied to any of the builds/tests/servers...so I will add it to the package.json config.